aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/Sample_Notebook_Template/title_of_your_product-Algorithm.ipynb

```diff
@@ -226,6 +226,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "sagemaker_session = sage.Session()\n",
```

Comment:
`@ignorable`: i know that it's already aliased as `sage` above, so it's out-of-scope for this review, but it seems the general convention is not to alias to `sage` and leave as `sagemaker`.

Reply:
Sure, ill keep this in mind next time.